app/src/webview/java/org/mozilla/focus/webview/SystemWebView.java

```diff
@@ -55,7 +55,7 @@
     public SystemWebView(Context context, AttributeSet attrs) {
         super(context, attrs);
 
-        client = new FocusWebViewClient(getContext().getApplicationContext());
+        client = new FocusWebViewClient(context);
```

Comment:
This is a risky change. Are you sure nothing is keeping a long-lived reference to this context?